src/Configuration.php

```diff
@@ -50,6 +51,11 @@
      */
     private $validator;
 
+    /**
+     * @var Constraint[]|null
+     */
+    private $constraints;
```

Comment:
We could configure the default value already here, so that the property would never be `null`.

Reply:
What do you think about renaming the property in order to align to the accessors' names (getter/setter) - rename to `validationConstraints`?

Reply:
Yes, totally agree. I will rename this var and the param in the setter function.